_data/reference.yml

```diff
@@ -183,45 +183,45 @@ toc:
 - title: Glossary
   section:
   - title: Annotations
-    path: /docs/user-guide/annotations/  
+    path: /docs/user-guide/annotations/
   - title: Daemon Sets
     path: /docs/admin/daemons/
-  - title: Deployments    
+  - title: Deployments
     path: /docs/user-guide/deployments/
   - title: Horizontal Pod Autoscaling
-    path: /docs/user-guide/horizontal-pod-autoscaling/    
+    path: /docs/user-guide/horizontal-pod-autoscaling/
   - title: Images
     path: /docs/user-guide/images/
   - title: Ingress Resources
-    path: /docs/user-guide/ingress/    
+    path: /docs/user-guide/ingress/
   - title: Jobs
-    path: /docs/user-guide/jobs/    
+    path: /docs/user-guide/jobs/
   - title: Labels and Selectors
     path: /docs/user-guide/labels/
   - title: Names
     path: /docs/user-guide/identifiers/
   - title: Namespaces
     path: /docs/user-guide/namespaces/
-  - title: Network Policies 
-    path: /docs/user-guide/networkpolicies/    
+  - title: Network Policies
+    path: /docs/user-guide/networkpolicies/
   - title: Nodes
-    path: /docs/admin/node/    
+    path: /docs/admin/node/
   - title: Persistent Volumes
-    path: /docs/user-guide/persistent-volumes/    
+    path: /docs/user-guide/persistent-volumes/
   - title: Pet Sets
-    path: /docs/user-guide/petset/    
+    path: /docs/user-guide/petset/
   - title: Pods
     path: /docs/user-guide/pods/
   - title: Pod Security Policies
     path: /docs/user-guide/pod-security-policy/
   - title: Replica Sets
     path: /docs/user-guide/replicasets/
   - title: Replication Controller
-    path: /docs/user-guide/replication-controller/    
+    path: /docs/user-guide/replication-controller/
   - title: Resource Quotas
-    path: /docs/admin/resource-quota/    
-  - title: Scheduled Jobs
-    path: /docs/user-guide/scheduled-jobs/
+    path: /docs/admin/resource-quota/
+  - title: Cron Jobs
+    path: /docs/user-guide/cron-jobs/
```

Comment:
Not quite, this PR builds on top of #1506 which added ScheduledJob to `_data/guides.yml`. Previously it wasn't even available there. I've just checked against what we have in master going through `Reference > Glossary > Jobs` will get you to `Guides > Batch Jobs > Jobs`. Not sure what would you like to see there?